### DISCUSSION TITLE
Kafka support status

### DISCUSSION THREAD

**Discussion by AntonOfTheWoods** [celery/celery#7565] [General]
It looks like `kafka` is now beginning to be supported in `kombu` which, given my limited understanding, is a prereq to support in `celery`. The readme for `kombu` doesn't list `kafka` yet though, and I can only find references like "work to support kafka" in issues dating up to several years ago. 

I couldn't find `kafka` support on any of the milestones. Did I look wrong? Should I add a ticket?

**Comment by auvipy** [comment]
no you look right, no need to create ticket. already tickets are open

**Reply by AntonOfTheWoods** [reply]
Thanks, could you post the tickets so I can subscribe? Searching for "kafka" in this repo doesn't show any obvious tickets.
